worldengine/generation.py

```diff
@@ -212,7 +213,7 @@ def generate_world(w, step):
 
     # FIXME: create setters
     IrrigationSimulation().execute(w, seed)
-    TemperatureSimulation().execute(w, seed)
+    
```

Comment:
Ok, thanks for the clarification.
This is the change that minorly interferes with https://github.com/Mindwerks/worldengine/pull/146 , so you may have to redo this should https://github.com/Mindwerks/worldengine/pull/146 be merged first (which should be trivial).